src/resources/help/en_US/relnotes3.4.0.html

```diff
@@ -120,6 +120,9 @@
 <ul>
   <li>Support for boolArray and boolVector was added to xml code gen tools.</li>
   <li>Updated the "Creating a Release" section of the developer manual to describe the current process.</li>
+  <li>Add support for Qt6.  The build_visit option is '--qt6', which will build qt version 6.4.2, and also force building of VTK-9 if '--vtk' is also enabled.</li>
+  <li>Developer Changes 2</li>
```

Comment:
?

Reply:
Was I not clear enough in the comment? Or is your '?' something different? Can you clarify?

Reply:
Sorry - I am confused why you added "Developer Changes 2".